__tests__/get-tokens.spec.js

```diff
@@ -1,26 +1,23 @@
 /*
  * The integration test to test the whole business, with DB
  */
-require('dotenv').config()
+require('dotenv').config();
 const request = require('supertest');
-const server = require("../server/app");
+const server = require('../server/app');
 const { expect } = require('chai');
 const seed = require('./seed');
-const log = require('loglevel');
-const Transfer = require("../server/models/Transfer");
-const TrustRelationship = require("../server/models/TrustRelationship");
-const sinon = require("sinon");
-const chai = require("chai");
+const sinon = require('sinon');
+const chai = require('chai');
 chai.use(require('chai-uuid'));
 
 describe('GET tokens', () => {
   let bearerToken;
   let bearerTokenB;
 
-  before( async () => {
-
+  beforeEach(async () => {
```

Comment:
Yes, I think in the same way, to make the tests independent, and I also are building the test in the way similar with your wat to add some `seed` or data-preparing function to create the data senerioir, take a look this PR:
https://github.com/Greenstand/treetracker-wallet-api/pull/192

Reply:
yeah, I saw in some places tests were depending on each other and it's a bad practice - makes making changes very complicated and generally it's error prone. Ideally, every unit test should create data for itself and each time data should be erased after the test, I think